index.html

```diff
@@ -9,6 +9,7 @@
     <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
 </head>
 <body>
```

Comment:
[OPTIONAL] There is white space on the top of your page. I think it would be better to get rid of it. 👍 Check the image below.
<img width="416" alt="Screenshot 2023-03-25 at 12 53 08 AM" src="https://user-images.githubusercontent.com/25887253/227648545-be939ef8-d571-4a65-a7e7-058b53d3a0a3.png">